src/Workspaces/Remote/ServiceHub/Services/ExtensionMethodImportCompletion/RemoteExtensionMethodImportCompletionService.cs

```diff
@@ -40,9 +38,13 @@ public RemoteExtensionMethodImportCompletionService(in ServiceConstructionArgume
             bool hideAdvancedMembers,
             CancellationToken cancellationToken)
         {
+            var stopwatch = SharedStopwatch.StartNew();
             return RunServiceAsync(solutionChecksum, async solution =>
             {
-                var document = solution.GetDocument(documentId)!;
+                var assetSyncTime = stopwatch.Elapsed;
```

Comment:
This is the solution-asset syncing caused delay I'm interested in but not being measured before.